.github/workflows/build.yml

```diff
@@ -137,3 +137,31 @@ jobs:
             DBUILD_DATE=${{ steps.date.outputs.DATE }}
             DBUILD_REPO_URL=https://github.com/openebs/dynamic-localpv-provisioner
             DBUILD_SITE_URL=https://openebs.io
+
+  ansible:
+    runs-on: ubuntu-latest
+    needs: ['unit-tests']
```

Comment:
This might be a typo, the unit test job is called 'unit-test'.
But, the localpv-e2e image build does not depend on the tests being run in the unit-test job.
I think this line is not needed here.